src/components/pages/public/RegisterPage.vue

```diff
@@ -1,12 +1,69 @@
 <template>
-  <div>
-    This is the register page.
-    <router-view></router-view>
+  <div class="register-page">
+    <div class="register-page__wrapper">
+      <h1 class="register-page__heading">Ahoy you!</h1>
+      <h2 class="register-page__subheading">Care to register?</h2>
+      <RegisterFirstCard />
+    </div>
   </div>
 </template>
 
 <script>
-export default {};
+import RegisterFirstCard from "@/components/RegisterFirstCard.vue";
+
+export default {
+  name: "RegisterFirstPage",
+  components: {
+    RegisterFirstCard,
+  },
+  data() {
+    return {
+      inputValue: "",
+    };
+  },
+  methods: {
+    setInput(event) {
+      this.inputValue = event.target.value;
+    },
+  },
+};
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+.register-page {
+  &__wrapper {
+    max-width: 624px;
+    padding: 2em 0.5em 0;
+    margin: 0 auto;
+  }
+
+  &__heading,
+  &__subheading {
+    @include font-heading--large;
+  }
+
+  &__subheading {
+    color: $text-light;
+    padding-bottom: 0.5em;
+  }
+
+  &__description-wrapper {
```

Comment:
some styles in this file (but not only this file) are not used anymore because code was moved to other components - it's important to remove unused styles, as it may influence your application size, so is the perceived performance  for your users.